cuda_core/cuda/core/experimental/_stream.py

```diff
@@ -89,7 +89,7 @@ def _init(obj=None, *, options: Optional[StreamOptions] = None):
         if obj is not None:
             if not hasattr(obj, "__cuda_stream__"):
                 raise ValueError
-            info = obj.__cuda_stream__
+            info = obj.__cuda_stream__()
```

Comment:
FYI, if we want to integrate cuda.core with any project today, we could consider a "soft break" here by trying method first and falling back to attribute if it's not a method.

Reply:
That sounds like a good idea. Do you want me to add that?

Reply:
Sure! Thanks!